sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/misc.scala

```diff
@@ -347,12 +347,12 @@ case class Murmur3Hash(children: Seq[Expression], seed: Int) extends Expression
       input: String,
       dataType: DataType,
       seed: String,
-      ctx: CodeGenContext): GeneratedExpressionCode = {
+      ctx: CodegenContext): ExprCode = {
     val hasher = classOf[Murmur3_x86_32].getName
-    def hashInt(i: String): GeneratedExpressionCode = inlineValue(s"$hasher.hashInt($i, $seed)")
-    def hashLong(l: String): GeneratedExpressionCode = inlineValue(s"$hasher.hashLong($l, $seed)")
-    def inlineValue(v: String): GeneratedExpressionCode =
-      GeneratedExpressionCode(code = "", isNull = "false", value = v)
+    def hashInt(i: String): ExprCode = inlineValue(s"$hasher.hashInt($i, $seed)")
+    def hashLong(l: String): ExprCode = inlineValue(s"$hasher.hashLong($l, $seed)")
+    def inlineValue(v: String): ExprCode =
```

Comment:
this can fit in one line now. no need to do it in this pr (maybe just update it in ur other one)